api/strategies/openidStrategy.js

```diff
@@ -57,7 +60,13 @@ async function setupOpenId() {
           let user = await User.findOne({ openidId: userinfo.sub });
 
           if (!user) {
-            user = await User.findOne({ email: userinfo.email });
+            if (!userinfo.email) {
```

Comment:
Thanks for the recent update. This caught my attention, there is some nesting here that is confusing. In general I'm [against nesting](https://www.martin-paucot.fr/blog/stop-nesting-your-code-4ng8) but it would make more sense to have this condition as a separate block above the user not being found at all

Reply:
I looked deeper and remove this print, because it most for more logs, not for important logic.

Reply:
@danny-avila ?